### ISSUE TITLE
World Not Regenerating - Lucky Blocks Compatibility Issue?

### ISSUE THREAD

**Issue by Kirrasio** [ScreamingSandals/BedWars#121]
After spawning in structures and entities, the world remains the same and does not reset after it has restarted. We are using SuperLuckyBlock plugin. This also appears to be an issue for other items. IE: Custom Denizen items that break blocks & cause explosions.

BedWars-0.2.6-SNAPSHOT-444
Paper Spigot version git-Paper-143 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)

Plugins (21): BedWars, Citizens, CMI, CrazyEnchantments, Denizen, FastAsyncWorldEdit, LibsDisguises, LuckPerms, MoarBows, Multiverse-Core, MythicMobs, PhatLoots, PlaceholderAPI, PocketGames, ProtocolLib, Shopkeepers, SuperLuckyBlock, Vault, VoidGenerator, WorldEdit, WorldGuard

**Comment by Misat11** [comment]
Did you try to disable Lucky Blocks if the problem is still present?

**Comment by Kirrasio** [comment]
Hey there, I'm sorry I haven't updated this, I have been super busy. I've downloaded the latest and I'm going to give some updates on this. Worldedit pastes are not regenerating either, I'm using fawe paste functionality to test it. I pulled everything I had installed out except these plugins: bedwars, cmi, fawe, luckypermisisons, placeholderapi, protocollib, vault, voidgenerator, worldborder and worldguard and the problem still persists. Seems anything pasted in from different plugins persist including entities.

Version: git-Spigot-2040c4c-893ad93 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)

BedWars-0.2.7-SNAPSHOT-500.jar
